platforms/minigame/common/engine/AssetManager.js

```diff
@@ -21,32 +21,33 @@ if (sys.platform === sys.Platform.BAIDU_MINI_GAME) {
 function downloadScript (url, options, onComplete) {
     if (REGEX.test(url)) {
         onComplete && onComplete(new Error('Can not load remote scripts'));
-    }
-    else {
+    } else {
         //TODO: Can't load scripts dynamically on Taobao platform
         if (sys.platform !== sys.Platform.TAOBAO_CREATIVE_APP) {
-            require('../../../' + url);
+            if (sys.platform === sys.Platform.TAOBAO_MINI_GAME) {
+                __taobaoRequire(`../../../${url}`);
```

Comment:
Is `__taobaoRequire ` an interface provide by taobao minigame?

Reply:
require has two versions of implementation, one for Taobao and one for the in-engine